nextflow_schema.json

```diff
@@ -982,28 +982,43 @@
+                    "help_text": "Turns off damage calculation to compute DNA damage profiles."
                 },
-                "damageprofiler_length": {
+                "damage_calculation_tool": {
+                    "type": "string",
+                    "default": "damageprofiler",
+                    "enum": ["damageprofiler", "mapdamage"],
+                    "fa_icon": "fas fa-tools",
+                    "description": "Specify the tool to use for damage calculation.",
+                    "help_text": "Specify the tool to be used for damage calculation. DamageProfiler is generally faster than mapDamage2, but the latter has an option to limit the number of reads used. This can significantly speed up the processing of very large files, where the damage estimates are already accurate after processing only a fraction of the input. Options: damageprofiler, mapdamage. By default, DamageProfiler is used."
+                },
+                "damage_calculation_yaxis": {
+                    "type": "number",
+                    "default": 0.3,
+                    "description": "Specify the maximum misincorporation frequency that should be displayed on damage plot. Set to 0 to 'autoscale'.",
+                    "help_text": "Specifies what the maximum misincorporation frequency should be displayed as, in the damage plot. This is set to 0.30 (i.e. 30%) by default as this matches the popular [mapDamage2.0 program](https://ginolhac.github.io/mapDamage/).\n\n> Modifies DamageProfiler parameter: -yaxis_dp_max or mapDamage2 parameter: --ymax",
+                    "fa_icon": "fas fa-ruler-combined"
+                },
+                "damage_calculation_damageprofiler_length": {
                     "type": "integer",
                     "default": 100,
-                    "fa_icon": "fas fa-sort-amount-up",
                     "description": "Specifies the length filter for DamageProfiler.",
-                    "help_text": "> Modifies DamageProfile parameter: -l "
+                    "help_text": "> Modifies DamageProfiler parameter: -l",
+                    "fa_icon": "fas fa-sort-amount-down"
                 },
-                "damageprofiler_threshold": {
+                "damage_calculation_damageprofiler_threshold": {
                     "type": "integer",
                     "default": 15,
                     "description": "Specify number of bases of each read to consider for DamageProfiler calculations.",
-                    "fa_icon": "fas fa-ruler-horizontal",
-                    "help_text": "Specifies the length of the read start and end to be considered for profile generation in DamageProfiler.\n\n> Modifies DamageProfile parameter: -t\n"
+                    "help_text": "Specifies the length of the read start and end to be considered for profile generation in DamageProfiler.\n\n> Modifies DamageProfiler parameter: -t\n",
```

Comment:
I don't know what I meant with that, could you find it out and make it better?

Reply:
The span of the read which the misincorporations frequencies are calcualted from